github-release.rb

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env ruby
+
+if File.basename($PROGRAM_NAME) != 'rake'
+  require 'shellwords'
+  puts "bundle exec rake -f #{Shellwords.escape($PROGRAM_NAME)} #{Shellwords.shelljoin(ARGV)}"
+  exec "bundle exec rake -f #{Shellwords.escape($PROGRAM_NAME)} #{Shellwords.shelljoin(ARGV)}"
+end
+
+require 'rest-client'
+require 'json'
+
+task :release do
+  version_to_release = ENV['VERSION_TO_RELEASE']
+  commit = ENV["COMMIT"]
+  token = ENV["TOKEN"]
+
+  hash = {
+      tag_name: version_to_release,
+      target_commitish: commit,
+      name: "GoCD #{version_to_release}",
+      body: "Checkout the release notes at https://www.gocd.org/releases",
+      draft: false,
+      prerelease: false
+  }
+  release = RestClient.post("https://api.github.com/repos/gocd/gocd/releases", hash.to_json, {:accept => 'application/vnd.github.v3+json', :Authorization => "token #{token}"})
+  p JSON.parse(release)
+end
+
+task :bump_plugins do
```

Comment:
`bump_plugins` is not needed anymore, I think. We're not publishing anything that's in the repository.